src/Card/index.jsx

```diff
@@ -28,7 +31,10 @@ const Card = React.forwardRef(({
       })}
       ref={ref}
       tabIndex={isClickable ? '0' : '-1'}
-    />
+    >
+      {actionIcon ? <CardActionIcon actionIcon={actionIcon} variant="dark" /> : undefined}
```

Comment:
[curious] I'm wondering if it would make sense to have the consumer render a `Card.ActionIcon` sub-component in the `children` of `Card` to render this component, rather than relying on an `actionIcon` prop at the top-level `Card` component to be more in line with the other sub-components part of `Card`, e.g.

```jsx
<Card>
  <Card.ActionIcon />
</Card>
```